graph/src/data/query/error.rs

```diff
@@ -78,10 +105,46 @@ impl fmt::Display for QueryExecutionError {
                 f,
                 "Only a single top-level field is allowed in subscriptions"
             ),
+            QueryExecutionError::SubgraphParseError(s) => {
+                write!(f, "Failed to get subgraph ID from type: {}", s)
+            }
+            QueryExecutionError::BuildRangeTypeError => {
+                write!(f, "Range inputs must be an integer")
+            }
+            QueryExecutionError::BuildFilterError => write!(f, "Filter must by an object"),
+            QueryExecutionError::EntityAttributeError => {
+                write!(f, "Attribute does not belong to entity")
+            }
+            QueryExecutionError::ListTypesError() => {
+                write!(f, "List field contains inconsistent Value types")
```

Comment:
Users/clients don't know about `Value`, so we shouldn't mention the type here. Secondly, this is not about a `list field`, it's about the values in a list passed to a filter. Thirdly, we should report which filter this is about.

So:
```rust
ListTypesError(String)
```
Then, make the error message:
```rust
"Values passed to filter '{}' must be of the same type but are of different types"
```